src/main/kotlin/org/jetbrains/research/testspark/helpers/LLMHelper.kt

```diff
@@ -303,9 +303,15 @@ object LLMHelper {
             }
         }
 
+        val inappropriateModelSubstrings = mutableListOf("vision", "test", "preview")
+
```

Comment:
Do I understand correctly that this is a temporal solution? Or is it always the case that models tagged with these labels cannot be used to test generation?

Reply:
Such a conclusion is based on the generation experiments. 
`gpt-4o-test` -- the only model currently containing the "test" substring, on my machine, unfortunately, the result is sometimes unsuccessful.
`vision` models are models specialised for working with images, and we removed them from the list long time ago due to failed attempts to generate tests.
`preview` models are the only models from this list that can be kept. And maybe this is even a good decision, so that researchers can try out preview generation of subsequent models. I'll make the changes in the next commit.

More detailed description of the models here: https://platform.openai.com/docs/models

Reply:
+

Reply:
@pderakhshanfar, should we filter out the model in this case or preserve it?

Reply:
I don't see the `gpt-4o-test` in the OpenAI [documentation](https://platform.openai.com/docs/models). 

> on my machine, unfortunately, the result is sometimes unsuccessful.

@arksap2002, what do you mean by unsuccessful? Does it mean that it cannot pass the compilation feedback cycle?